src/includes/home/api-example.njk

```diff
@@ -68,7 +81,20 @@ request = Net::HTTP::Post.new(url)
 request["Accept"] = 'application/json'
 request["Content-Type"] = 'application/json'
 request["Authorization"] = 'Bearer <orbit-api-key>'
-request.body = "{\"activity\":{\"member\":{\"github\":\"user-github\",\"email\":\"user@email\"},\"description\":\"Getting Started with the Orbit Model\",\"link\":\"https://orbit-event.eventbrite.com/\",\"link_text\":\"See the event\",\"title\":\"Orbit Meetup Registration\",\"activity_type\":\"eventbrite:registration\",\"occurred_at\":\"2020-05-10\"}}"
+request.body = {
+    activity: {
+        description: 'Getting Started with the Orbit Model',
+        link: 'https://orbit-event.eventbrite.com/',
+        link_text: 'See the event',
+        title: 'Orbit Meetup Registration',
+        activity_type: 'eventbrite:registration',
+        occurred_at: '2020-05-10',
+        member: {
+            github: 'user-github',
+            email: 'user@email'
```

Comment:
@bencgreenberg Small tweak - `user@email.com` ?

Reply:
Yes, we can do that. The original snippets didn't have the `.com`. I wasn't sure if that was intentional. 🤷🏻